### PR TITLE
Add reconnect option and update setting connection options

### PR DESCRIPTION
**Pull request by zschwarz**

### CODE REVIEW

File: cli-paho-java/src/main/java/com/redhat/mqe/amc/Receiver.java [hunk 1]
```diff
@@ -64,6 +64,9 @@ public void startClient() throws MqttException {
             throw e;
         } catch (InterruptedException e) {
             e.printStackTrace();
+        } catch (Exception ex) {
+            ex.printStackTrace();
+            System.exit(1);
```
Comment (jdanekrh) [inline cli-paho-java/src/main/java/com/redhat/mqe/amc/Receiver.java]: why is this necessary? can't you let there be uncaught exception? it will stop jvm too, print stacktrace, and exit code will be 1

**Comment by jdanekrh** [comment]
@zschwarz please merge this when you are satisfied

File: cli-paho-java/src/main/java/com/redhat/mqe/amc/Receiver.java (hunk 1 above)
Comment (zschwarz) [reply cli-paho-java/src/main/java/com/redhat/mqe/amc/Receiver.java]: Without it, when receiver is connected and you shut down broker, receiver throws an error because of lost connection and it doesn't shutdown properly.

Comment (jdanekrh) [reply cli-paho-java/src/main/java/com/redhat/mqe/amc/Receiver.java]: So, the receiver stays running, does not shut down at all, in that case?

Comment (zschwarz) [reply cli-paho-java/src/main/java/com/redhat/mqe/amc/Receiver.java]: Yes, exactly.

Comment (jdanekrh) [reply cli-paho-java/src/main/java/com/redhat/mqe/amc/Receiver.java]: Ok, then ship it! (merge it, I mean)

Comment (zschwarz) [reply cli-paho-java/src/main/java/com/redhat/mqe/amc/Receiver.java]: Aye! Thanks.
